src/components/Placeholder.js

```diff
@@ -10,13 +10,17 @@ export type PlaceholderProps = CommonProps & {
   children: Node,
   /** props passed to the wrapping element for the group. */
   innerProps: { [string]: any },
-};
+} & State;
+
+export type State = {
+  cssPosition: 'relative' | 'absolute',
+}
 
-export const css = () => ({
+export const css = ({ cssPosition }: State) => ({
   color: colors.neutral50,
   marginLeft: spacing.baseUnit / 2,
   marginRight: spacing.baseUnit / 2,
-  position: 'absolute',
+  position: cssPosition || 'absolute',
```

Comment:
the base style for placeholder and singlevalue set the position value to 'absolute' which neutralises the core premise of this implementation. I don't if this is the best way to go about fixing this for *just the instance of singleValue and placeholder we use in ValueSpacer, and I'd like to discuss this and its alternatives in the context of not complicating user customisations.